zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/ServiceStoreImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.github.onsdigital.zebedee.service;
+
+import com.github.onsdigital.zebedee.model.PathUtils;
+import com.github.onsdigital.zebedee.model.ServiceAccount;
+import com.github.onsdigital.zebedee.util.serialiser.JSONSerialiser;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ServiceStoreImpl implements ServiceStore {
+
+    private final Path rootLocation;
+
+    private final JSONSerialiser<ServiceAccount> jsonSerialise;
+
+    private final static String JSON_EXTENSION = ".json";
+
+    public ServiceStoreImpl(Path rootLocation) {
+        this.rootLocation = rootLocation;
+        this.jsonSerialise = new JSONSerialiser<>(ServiceAccount.class);
+    }
+
+    @Override
+    public ServiceAccount get(String id) throws IOException {
+        ServiceAccount service = null;
+        final Path path = getPath(id);
+        if (Files.exists(getPath(id))) {
+            try (InputStream input = Files.newInputStream(path)) {
+                service = jsonSerialise.deserialiseQuietly(input, path);
+            }
+        }
+        return service;
+    }
+
+    @Override
+    public ServiceAccount store(String token, InputStream service) throws IOException {
+        final Path path = rootLocation.resolve(token + JSON_EXTENSION);
+        ServiceAccount object = jsonSerialise.deserialiseQuietly(service, path);
```

Comment:
Do you need to check if it already exists?

Reply:
Good point 👍, I added a check in